app/assets/stylesheets/playtime.scss

```diff
@@ -71,6 +72,16 @@ h3 {
   float: right;
 }
 
+.alert {
```

Comment:
It looks like this (L75-84) override some of the Bootstrap default styling. Is that what you intended? It looks like these were written before the switch to Bootstrap. In particular, the extra space under the message and the width being slightly different from the title/hr width look a little off to me.

For comparison:

**This PR:**
![Overridden Bootstrap styles](https://user-images.githubusercontent.com/814638/28832246-fec8a14c-76a9-11e7-9f71-a7ef95e13266.png)

**Default Bootstrap** (+ vertical padding)
![Default bootstrap](https://user-images.githubusercontent.com/814638/28832276-1caf7b90-76aa-11e7-8799-88694b364cc4.png)